docs/04_web/03_design-systems/03_preview.md

```diff
@@ -9,7 +9,72 @@ tags:
   - preview
 ---
 
+## Introduction
+
+The Design System Configurator you see below, will be the tool we use to configure the design tokens and determine the look that's best suited for our application.
+
+On the left hand side, is the [Editor](../../../web/design-systems/preview/#editor), where we can make all the modifications we want. On the right side, we can see the [Preview](../../../web/design-systems/preview/#preview) with the results of those changes.
+
+Try changing the configurations and observe the reflected changes on the right.
+
 <design-system-editor>
```

Comment:
I tried that but while it is extending the width it also ends up throwing the layout off. Especially if we scroll down. I thought rather than adding more custom css to fix that, I'd leave it alone for now. It's not ideal but what do you think @nick-genesis 
![image](https://github.com/genesiscommunitysuccess/docs/assets/102529447/967c5a85-cd22-4311-b820-38fd8fb49822)